packages/daimo-userop/dist/src/daimoAccountBuilder.d.ts

```diff
@@ -0,0 +1,18 @@
+import { UserOperationBuilder } from "userop";
+import { UserOperationMiddlewareFn } from "userop";
+import { SigningCallback } from "./util";
+import "@ethersproject/shims";
+import "text-encoding-polyfill";
+export declare class DaimoAccountBuilder extends UserOperationBuilder {
```

Comment:
recommend `.gitignore`ing `dist`

Reply:
would like to, yeah, see https://github.com/daimo-eth/daimo/pull/25#discussion_r1225900274

Reply:
i see, makes sense!